addons/block_code/serialization/block_script_serialization.gd

```diff
@@ -182,6 +189,17 @@ func _get_parent_node_property_info(property_name: String) -> Dictionary:
 	return {}
 
 
+func _get_resource_block_definition(block_name: String, file_path: String) -> BlockDefinition:
+	if not block_name.begins_with("get_resource_"):
+		return null
```

Comment:
This doesn't appear to use the name embedded in the block name at all. What was the use of that?

Reply:
I set it as "get_resource_file_path" in the fixed commit.